src/domain/storage/storage-bucket/storage.bucket.service.ts

```diff
@@ -300,10 +296,24 @@ export class StorageBucketService {
   private validateMimeTypes(
     storageBucket: IStorageBucket,
     mimeType: string
-  ): boolean {
+  ): void {
```

Comment:
I would not do that. `Validate` in software development is used as a method to validate the correctness of a condition and return a `boolean` value. 

This altered implementation is a hybrid between `validate` and `tryGet...`. It is unclean, confusing, and not fit-for-purpose in my opinion. Also, if it were to stay, the actual return type is `void | never`, as an exception is thrown inside the method under certain conditions.

Reply:
It is consistent with the rest at least, so I will just merge it...